department-of-veterans-affairs/abd-vro
# Brostk/3031

## What was the problem?
Scheduled SecRel runs are failing due to missing the container image names set to be signed and published.

Associated tickets or Slack threads:
- #3031 

## How does this fix it?[^1]
PUBLISH_IMAGES and RUN_SECREL flags are now being set from the gate check due to having added a condition to an if/else block which previously did not support "schedule" event types.

## How to test this PR
- Wait for a scheduled SecRel run to be performed and examine the results of the run.  Images should be signed and published.


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
